shopify/wishlist-template/web/routes/index.tsx

```diff
@@ -32,11 +32,11 @@ export default function () {
     <Page title="Wishlist template">
       <Layout>
         <Layout.Section>
-          <Text>
+          <Text as="p">
             The following is an example of updating the default update
             frequency. This is a field on the `shopifyShop` model.
           </Text>
-          <Form>
+          <Form onSubmit={() => {}}>
```

Comment:
Two comments/questions:
- Should we be using `AutoForm` here?
- If we don't use `AutoForm` I feel like we should be submitting through the form onSubmit, not the Button onClick

Reply:
Fixed!